misc/test/examples_test.go

```diff
@@ -67,6 +67,32 @@ func TestExamples(t *testing.T) {
 				expectHelloWorld(t, stack.Outputs["publicIP"])
 			},
 		}),
+		base.With(integration.ProgramTestOptions{
+			Dir:       path.Join(cwd, "..", "..", "azure-ts-functions"),
+			SkipBuild: true,
+			Config: map[string]string{
+				"azure:environment": azureEnviron,
+			},
+			ExtraRuntimeValidation: func(t *testing.T, stack integration.RuntimeValidationStackInfo) {
+				func(t *testing.T, output interface{}) bool {
+					endpoint, ok := output.(string)
+					if !assert.True(t, ok, fmt.Sprintf("expected `%s` output", output)) {
+						return false
+					}
+					resp, err := http.Get(endpoint)
+					if !assert.NoError(t, err) {
+						return false
+					}
+					defer resp.Body.Close()
+					body, err := ioutil.ReadAll(resp.Body)
+					if !assert.NoError(t, err) {
+						return false
+					}
+					return assert.Contains(t, string(body), "Greetings from Azure Functions!")
+				}(t, stack.Outputs["endpoint"])
```

Comment:
Along the lines of #139, let's try to share more of the HTTP request functionality across tests and generalize that as needed as we expand test coverage.